src/demo/index.ejs

```diff
@@ -7,8 +7,17 @@
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="format-detection" content="telephone=no">
-
-    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600" rel="stylesheet">
+    <% if (process.env.NODE_ENV !== "development"){ %>
+    <meta http-equiv="Content-Security-Policy" content="
+      default-src 'self' https://assets.onfido.com;
+      script-src 'unsafe-eval' 'self' https://www.woopra.com https://assets.onfido.com;
+      style-src 'self' 'unsafe-inline' https://assets.onfido.com;
+      connect-src *.onfido.com <%=process.env.DESKTOP_SYNC_URL%> https://www.woopra.com;
```

Comment:
I don't think this will work when we switch protocol to `ws` or `wss`

Reply:
you are right, I just tested it.
I actually added it because of `wss`, but it's only because of your comment that I realised the link doesn't have `wss`.
Great catch.